packages/grid/_modules_/grid/models/api/gridRowsMetaApi.ts

```diff
@@ -11,4 +11,11 @@ export interface GridRowsMetaApi {
    * @ignore - do not document.
    */
   unstable_getRowHeight: (id: GridRowId) => number;
+  /**
+   * Updates the height of a row.
```

Comment:
Only a detail but the height here is for the row element. There might be additional margins and the detail panel which makes weight in `rowsMeta.positions` but shouldn't be used for the row element.

```suggestion
   * Updates the base height of a row.
```